extensions/python/PythonDependencyInstaller.cpp

```diff
@@ -136,6 +136,7 @@ void PythonDependencyInstaller::evalScript(std::string_view script) {
   }
 
   OwnedDict bindings = OwnedDict::create();
+  bindings.put("__builtins__", OwnedObject(PyImport_ImportModule("builtins")));
```

Comment:
Can you please explain what's happening here?

Reply:
Unfortunately  I cannot precisely explain what was the issue here, but it seems that the python builtin variables and functions need to be added in the python bindings provided for the PyEval_EvalCode to evaluate the `sys.path` modification script to be executed. It seems that in some platform+python version combinations it is not needed as the python builtins are available by default, but other times it needs to be specified explicitly.